_posts/en/newsletters/2021-06-02-newsletter.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 'Bitcoin Optech Newsletter #151'
+permalink: /en/newsletters/2021/06/02/
+name: 2021-06-02-newsletter
+slug: 2021-06-02-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal to change Bitcoin Core's
+transaction selection algorithm for miner block templates to slightly
+increase miner profitability and give fee bumping users more
+collective leverage.  Also included are our regular sections describing
+software releases and release candidates, plus notable changes to popular
+Bitcoin infrastructure software.
+
+## News
+
+- **Candidate Set Based (CSB) block template construction:** Mark
+  Erhardt [posted][erhardt post] to the Bitcoin-Dev mailing list about
+  an [analysis][es analysis] he and Clara Shikhelman performed on an
+  alternative transaction selection algorithm for miners.  Bitcoin's
+  consensus rules enforce that no transaction can be included in a block
+  unless all of its unconfirmed ancestors are also included earlier in
+  that same block.  Bitcoin Core addresses this constraint by treating
+  each transaction with unconfirmed ancestors as if it contained both
+  the fees and the size of those ancestors.  For example, transaction B
+  depends on unconfirmed transaction A, so Bitcoin Core adds together
+  the fees paid by both transactions and divides them by the combined
```

Comment:
Since you're introducing this hypothetical scenario here, then perhaps a conditional would be clearer:

```suggestion
  the fees and the size of those ancestors.  For example, if transaction B
  depends on unconfirmed transaction A, then Bitcoin Core would add together
  the fees paid by both transactions and divide them by the combined
```